st2common/st2common/openapi.yaml.j2

```diff
@@ -4115,9 +4122,124 @@ definitions:
+          id:
+            type: string
+            description: Unique identifier for the action alias.
+          ref:
+            type: string
+            description: |
+              System computed user friendly reference for the alias. Provided
+              value will be overridden by computed value.
+          uid:
+            type: string
+          name:
+            type: string
+            description: Name of the action alias.
+          pack:
+            type: string
+            description: The content pack this actionalias belongs to.
+          description:
+            type: string
+            description: Description of the executed alias.
+            default: ""
+          enabled:
+            type: boolean
+            description: Flag indicating if action alias is enabled.
+            default: true
+          action_ref:
+            type: string
+            description: Reference to the aliased action
+          formats:
+            type: array
+            description: Possible parameter format.
+            items:
+              oneOf:
+              - type: string
+              - type: object
+                description: Matched alias formats
+                properties:
+                  representation:
+                    type: array
+                    description: Alias format representations
+                    items:
+                      type: string
+                  match_multiple:
+                    type: boolean
+                    optional: true
+                  display:
+                    type: string
+                    description: Display string of alias format
+          ack:
+            type: object
+            description: Acknowledgement message format.
+            properties:
+              enabled:
+                type: boolean
+              format:
+                type: string
+              extra:
+                type: object
+              append_url:
+                type: boolean
+          result:
+            type: object
+            description: Execution message format.
+            properties:
+              enabled:
+                type: boolean
+              format:
+                type: string
+              extra:
+                type: object
+          extra:
+            type: object
+            description: Extra parameters, usually adapter-specific
+      execution:
+        type: object
+        description: Execution result
+        properties:
+          id:
+            type: string
+            description: Execution identifier
+          start_timestamp:
+            type: string
+            description: Execution started timestamp
+            format: datetime
+          status:
+            type: string
+            description: Execution status
+            enum: ['requested', 'scheduled', 'delayed', 'running', 'succeeded', 'failed', 'timeout', 'abandoned', 'canceling', 'canceled', 'pending', 'pausing', 'paused', 'resuming']
```

Comment:
Should be able to utilize `{{ LIVEACTION_STATUSES }}` constant here (there should be same usage examples of that in this file).

Reply:
Might also be able to reference existing `Execution` object definition?

Reply:
Yes, I think I can simply reference `Execution` here.